openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -420,9 +420,11 @@ def request_password_change(email, is_secure):
     if form.is_valid():
         # Generate a single-use link for performing a password reset
         # and email it to the user.
+        from openedx.core.djangoapps.theming.helpers import get_current_request
```

Comment:
Nit: Is it necessary for this to be an inline import?

Reply:
Not at all, just a left over!